spec/aws/subnet.go

```diff
@@ -0,0 +1,8 @@
+package aws
+
+type Subnet struct {
+	//The name of the subnet
+	Name string `json:"name" yaml:"name"`
+	//The name of the route table associated to the subnet
+	RouteTableName string `json:"routeTableName" yaml:"routeTableName"`
```

Comment:
I am not into this topic. Where does this name come from? Kubernetesd has to provide this. Based on what mechanism does it do this?

Reply:
The name of the subnet and the route table are both set by `aws-terraform` ([here](https://github.com/giantswarm/aws-terraform/blob/master/resources/terraforms/vpc/vpc.tf#L30) and [here](https://github.com/giantswarm/aws-terraform/blob/master/resources/terraforms/vpc/vpc-subnet-worker.tf) ). I am not aware of the interaction with `kubernetesd` here.

Reply:
The route table names are predictable from the installation name. These would need to be set in `kubernetesd` but we have access to the install name so that is doable.

However, do we need to create a route per subnet? There is a subnet per AZ so we'd need 2 now and maybe 3 later. So I think this should be a string slice.

Reply:
OK I missed the slice with the private subnets. That looks correct to me.